src/services/kokkos/KokkosProfilingSymbols.cpp

```diff
@@ -197,6 +197,7 @@ __attribute__((weak)) void kokkosp_end_deep_copy() {
 namespace cali {
   extern Kokkos::Tools::Experimental::EventSet get_event_set() {
     Kokkos::Tools::Experimental::EventSet my_event_set;
+    memset(&my_event_set, 0, sizeof(my_event_set)); // zero any pointers not set here
```

Comment:
@crtrott Would this be a good way to initialize _all_ event set pointers with `nullptr` (also with regard to possible future extensions in the profiling interface) ? Or should we init all of them explicitly ?

Reply:
seems reasonable.